### ISSUE TITLE
V0.71 crash when opening application

### ISSUE THREAD

**Issue by myself12345** [segler-alex/RadioDroid#477]
Android 7.0.
V0.71  crash when :
- Playing on notification mode (main app closed)
- then  try to switch to the app from notification or from home icon.
- no crash if main app is not closed
No  problem with V0.7

**Comment by morckx** [comment]
Thanks for the report! Can you find something like this in your logs?
``` 
Not allowed to start service Intent { cmp=net.programmierecke.radiodroid2/.PlayerService }: app is in background
```
Otherwise, please provide a log from logcat (see https://android.gadgethacks.com/how-to/android-basics-capture-logcat-for-detailed-bug-reports-0167774/ ).

**Comment by myself12345** [comment]
Hello,
Here is is the logcat ( extract with adb logcat | grep radio)


05-10 13:46:51.732 10635 10635 E AndroidRuntime: Process: net.programmierecke.radiodroid2, PID: 1
0635
05-10 13:46:51.732 10635 10635 E AndroidRuntime: java.lang.RuntimeException: Unable to start acti
vity ComponentInfo{net.programmierecke.radiodroid2/net.programmierecke.radiodroid2.ActivityMain}:
 java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context androi
d.content.Context.getApplicationContext()' on a null object reference
05-10 13:46:51.732 10635 10635 E AndroidRuntime:        at net.programmierecke.radiodroid2.Fragme
ntPlayer$LifecycleCallbacks.onActivityStarted(FragmentPlayer.java:353)
05-10 13:47:11.838 11880 11880 E AndroidRuntime: Process: net.programmierecke.radiodroid2, PID: 1
1880
05-10 13:47:11.838 11880 11880 E AndroidRuntime: java.lang.RuntimeException: Unable to start acti
vity ComponentInfo{net.programmierecke.radiodroid2/net.programmierecke.radiodroid2.ActivityMain}:
 java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context androi
d.content.Context.getApplicationContext()' on a null object reference
05-10 13:47:11.838 11880 11880 E AndroidRuntime:        at net.programmierecke.radiodroid2.Fragme
ntPlayer$LifecycleCallbacks.onActivityStarted(FragmentPlayer.java:353)
05-10 13:47:30.289 11955 11955 E AndroidRuntime: Process: net.programmierecke.radiodroid2, PID: 1
1955
05-10 13:47:30.289 11955 11955 E AndroidRuntime: java.lang.RuntimeException: Unable to start acti
vity ComponentInfo{net.programmierecke.radiodroid2/net.programmierecke.radiodroid2.ActivityMain}:
 java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context androi
d.content.Context.getApplicationContext()' on a null object reference
05-10 13:47:30.289 11955 11955 E AndroidRuntime:        at net.programmierecke.radiodroid2.Fragme
ntPlayer$LifecycleCallbacks.onActivityStarted(FragmentPlayer.java:353)
05-10 14:10:30.579 14436 14436 E AndroidRuntime: Process: net.programmierecke.radiodroid2, PID: 1
4436
05-10 14:10:30.579 14436 14436 E AndroidRuntime: java.lang.RuntimeException: Unable to start acti
vity ComponentInfo{net.programmierecke.radiodroid2/net.programmierecke.radiodroid2.ActivityMain}:
 java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context androi
d.content.Context.getApplicationContext()' on a null object reference
05-10 14:10:30.579 14436 14436 E AndroidRuntime:        at net.programmierecke.radiodroid2.Fragme
ntPlayer$LifecycleCallbacks.onActivityStarted(FragmentPlayer.java:353)
05-10 14:10:49.920 14507 14507 E AndroidRuntime: Process: net.programmierecke.radiodroid2, PID: 1
4507
05-10 14:10:49.920 14507 14507 E AndroidRuntime: java.lang.RuntimeException: Unable to start acti
vity ComponentInfo{net.programmierecke.radiodroid2/net.programmierecke.radiodroid2.ActivityMain}:
 java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context androi
d.content.Context.getApplicationContext()' on a null object reference
05-10 14:10:49.920 14507 14507 E AndroidRuntime:        at net.programmierecke.radiodroid2.Fragme
ntPlayer$LifecycleCallbacks.onActivityStarted(FragmentPlayer.java:353)
05-10 14:11:14.412 14565 14565 E AndroidRuntime: Process: net.programmierecke.radiodroid2, PID: 1
4565
05-10 14:11:14.412 14565 14565 E AndroidRuntime: java.lang.RuntimeException: Unable to start acti
vity ComponentInfo{net.programmierecke.radiodroid2/net.programmierecke.radiodroid2.ActivityMain}:
 java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context androi
d.content.Context.getApplicationContext()' on a null object reference
05-10 14:10:30.579 14436 14436 E AndroidRuntime:        at net.programmierecke.radiodroid2.Fragme
ntPlayer$LifecycleCallbacks.onActivityStarted(FragmentPlayer.java:353)
05-10 14:10:49.920 14507 14507 E AndroidRuntime: Process: net.programmierecke.radiodroid2, PID: 1
4507
05-10 14:10:49.920 14507 14507 E AndroidRuntime: java.lang.RuntimeException: Unable to start acti
vity ComponentInfo{net.programmierecke.radiodroid2/net.programmierecke.radiodroid2.ActivityMain}:
 java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context androi
d.content.Context.getApplicationContext()' on a null object reference
05-10 14:10:49.920 14507 14507 E AndroidRuntime:        at net.programmierecke.radiodroid2.Fragme
ntPlayer$LifecycleCallbacks.onActivityStarted(FragmentPlayer.java:353)
05-10 14:11:14.412 14565 14565 E AndroidRuntime: Process: net.programmierecke.radiodroid2, PID: 1
4565
05-10 14:11:14.412 14565 14565 E AndroidRuntime: java.lang.RuntimeException: Unable to start acti
vity ComponentInfo{net.programmierecke.radiodroid2/net.programmierecke.radiodroid2.ActivityMain}:
 java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context androi
d.content.Context.getApplicationContext()' on a null object reference
05-10 14:11:14.412 14565 14565 E AndroidRuntime:        at net.programmierecke.radiodroid2.Fragme
ntPlayer$LifecycleCallbacks.onActivityStarted(FragmentPlayer.java:353)
05-10 14:11:31.993 14627 14627 E AndroidRuntime: Process: net.programmierecke.radiodroid2, PID: 1
4627
05-10 14:11:31.993 14627 14627 E AndroidRuntime: java.lang.RuntimeException: Unable to start acti
vity ComponentInfo{net.programmierecke.radiodroid2/net.programmierecke.radiodroid2.ActivityMain}:




[logcat.txt](https://github.com/segler-alex/RadioDroid/files/3167418/logcat.txt)

**Comment by morckx** [comment]
Thanks for the report! 

That bug is already fixed here 8f7e69450f6a768c22fbbd7f7d2d7b499585a336
The fix was merged yesterday, so I think it will be resolved in the next beta.

**Comment by myself12345** [comment]
I think indeed issue #468  is the same. 
Thank you for your very good app.

**Comment by morckx** [comment]
Can be closed, I guess. Fixed via #469.

**Comment by myself12345** [comment]
issue fixed with V0.72
thank you
